cristin-person/src/test/java/no/unit/nva/cristin/person/model/cristin/CristinPersonTest.java

```diff
@@ -0,0 +1,76 @@
+package no.unit.nva.cristin.person.model.cristin;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.IOException;
+import java.nio.file.Path;
+import no.unit.nva.cristin.person.model.nva.Person;
+import nva.commons.core.JsonUtils;
+import nva.commons.core.ioutils.IoUtils;
+import org.junit.jupiter.api.Test;
+
+public class CristinPersonTest {
```

Comment:
Generally, use the "shouldX" formulation for test names…it is clear what is tested. I assume these tests will be deleted anyway once the relevant handler is in place, but it is good practice to use the common naming format.